src/lavatory/utils/artifactory.py

```diff
@@ -54,13 +42,41 @@ def list(self):
         data = raw_data.json()
         LOG.debug('Storage info data: %s', data)
         for repo in data["repositoriesSummaryList"]:
-            if repo["repoKey"] == "TOTAL":
+            if repo['repoKey'] == "TOTAL":
                 continue
 
-            repos[repo["repoKey"]] = repo
+            repos[repo['repoKey']] = repo
 
         return repos
 
+    def get_statistics(self):
+        """Get statistics of a repo.
+            Examples: lavatory stats --repo docker-local
+
+        Returns:
+            str.
+        """
+        storage = self.list()
+        if self.repo_name:
+            if not storage.get(self.repo_name):
+                LOG.error('Repo %s does not exist!', self.repo_name)
+                return False
+            keys = [self.repo_name]
+        else:
+            keys = storage.keys()
+
+        for repository in keys:
+            repo = storage.get(repository)
+            LOG.info('*' * 25)
```

Comment:
I think this logging logic should be part of the command and not the `Artifactory` class. If someone was to import this class, they should be able to do the formatting how they want and this just returns the appropriate data. If you disagree, we can discuss, It just seems like it would make more sense for the command to handle the output and make it easier to test

Reply:
Sure I can take it outside.